docs/clients/dovecot.txt

```diff
@@ -36,7 +36,7 @@ Configuring Dovecot to work with Virtual Exim
 
       driver = mysql
       connect = host=/var/run/mysqld/mysqld.sock dbname=vexim user=vexim password=CHANGE
-      default_pass_scheme = CRYPT
+      default_pass_scheme = SHA512-CRYPT #BLF-CRYPT (<- this is for bcrypt, only in *BSD!) #PLAIN
```

Comment:
Do you know of a case where simply `CRYPT` would not work with our setup? I couldn't quickly find a source which would list which ciphers are available in which distributions, but I suspect it might be quite uniform, in which case we could just use tell software to use the crypt() implementation provided by the OS.